libsplinter/src/biome/rest_api/actix/login.rs

```diff
@@ -36,6 +37,7 @@ use super::super::resources::credentials::UsernamePassword;
 ///   }
 pub fn make_login_route(
     credentials_store: Arc<SplinterCredentialsStore>,
+    refresh_token_store: Arc<dyn RefreshTokenStore>,
```

Comment:
Either these lines with the refresh token store, or the login file itself, needs the 'biome-refresh-tokens` feature guard.

Reply:
Guarded where it's used it Biome Rest Resource provider